components/camel-xmlsecurity/src/test/java/org/apache/camel/dataformat/xmlsecurity/TestHelper.java

```diff
@@ -58,8 +57,7 @@
         + "</france>"
         + "</ns1:cheesesites>";
     
-    protected static final String XML_FRAGMENT = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>"
```

Comment:
XMLUnit 2.0 detects differences inside of the document's prolog (see: https://github.com/xmlunit/user-guide/wiki/Migrating-from-XMLUnit-1.x-to-2.x#changed-defaults-between-1x-and-2x) so I had to remove the xml-declaration to make the tests pass.